cloudsync/sync/manager.py

```diff
@@ -712,8 +714,11 @@ def handle_cloud_file_not_found_error(self, changed, sync, synced):
             return FINISHED
 
         # parent presumably exists
+        # why are we using the changed side path without translating it to local?
+        # why are we marking the folder as changed on the changed side?
+        # This won't do anything if we don't change the sync path or sync hash, btw...
```

Comment:
@earthman1 - I left these comments in as a remonder, is there anything actionable left to do here?

Reply:
The questions refer to the fact that this is specifically handling *cloud* file not found error, and we are addressing the changed side without respect for whether that is the cloud side or local side. What does it mean to mark the parent folder changed on one side or the other, if it is always the cloud side that has the file not found? I did not fully answer the question to my own satisfaction, but I felt it was an important consideration, or the basis of a test that introduces a cloud file not found error, once with changed local and once with changed remote, to ensure it handles all the cases